docs/troubleshooting.md

```diff
@@ -85,7 +90,8 @@ can be configured to inspect the data being processed by the Collector.
 
 For live troubleshooting purposes consider leveraging the `logging` exporter,
 which can be used to confirm that data is being received, processed and
-exported by the Collector.
+exported by the Collector. Note that for Windows today the `logging` exporter
```

Comment:
If this is tracked in some issue I think we should link it here

Reply:
Looks like it was buried in a separate issue -- broken out now: https://github.com/open-telemetry/opentelemetry-collector/issues/4086

Reply:
Added to line 94